WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -589,7 +591,9 @@ private extension AuthenticationManager {
     /// without Jetpack in the site discovery flow
     ///
     func jetpackErrorUI(for siteURL: String, with matcher: ULAccountMatcher, in navigationController: UINavigationController) -> UIViewController {
-        let viewModel = JetpackErrorViewModel(siteURL: siteURL, onJetpackSetupCompletion: { [weak self] authorizedEmailAddress in
+        let viewModel = JetpackErrorViewModel(siteURL: siteURL,
+                                              siteCredentials: nil,
```

Comment:
The site discovery flow is an addition that lets users enter a site address they want to log in to if they end up in an empty store picker (part of the improvements for successful logins: pe5sF9-mz-p2). I updated the comments to add a link to the project thread to avoid confusion in the future: c65f4c0b220219b8440940e1fe5a8c4716d2cd36.

The site credentials are nil here because the user follows the WP.com login flow to reach this state, so we don't have site credentials yet.

Reply:
Thanks for the explanation, TIL about the site discovery! 😮 The "Enter Your Store Address" CTA is exactly what I implemented today (just as a hack to test out the flow, screenshot below), I was too used to the bottom CTAs only to restart the flow 😆 glad to know that it's already implemented!

<img src="https://user-images.githubusercontent.com/1945542/192979533-f3ebd6ca-95fe-4f96-8e71-dd97e5a1d1b7.png" width="300" />